code/game/objects/structures/fires.dm

```diff
@@ -185,7 +185,8 @@
 			user.put_in_hands(removing)
 			if(lit == FIRE_LIT)
 				visible_message(SPAN_DANGER("\The [user] fishes \the [removing] out of \the [src]!"))
-				user.fire_act(return_air(), last_fuel_burn_temperature, 500)
+				// Uncomment this when there's a way to take stuff out of a kiln or oven without setting yourself on fire.
+				//user.fire_act(return_air(), last_fuel_burn_temperature, 500)
```

Comment:
could honestly just try calling `removing.try_burn_wearer(user)`? but maybe that'll happen anyways. honestly i kind of think holding a hot object is punishing enough for removing one from a lit kiln as it is